apps/extension/client/src/initiateEditor.ts

```diff
@@ -270,7 +282,7 @@ const machine = createMachine(
           ),
       onDisplayedMachineUpdatedListener:
         ({ extensionContext, languageClient }) =>
-        (sendBack) =>
+        (sendBack) => {
           registerDisposable(
```

Comment:
the return values from both of those `registerDisposable` calls here should be returned. This would look smth like this:
```ts
const dispose1 = registerDisposable(...)
const dispose2 = registerDisposable(...)
return () => {
  dispose1()
  dispose2()
}
```